qa/rpc-tests/wallet.py

```diff
@@ -119,6 +141,7 @@ def run_test (self):
         assert_equal(self.nodes[0].getbalance(), Decimal('19.99900000'))
 
         # Sendmany 10 BTC
```

Comment:
I am not sure which is preferred, but the code should only have either a comment or the `print`, imo.